requirements.txt

```diff
@@ -10,3 +10,4 @@ kombu==4.2.1
 marshmallow==3.0.0b14
 pyyaml==3.13  # from xivo-lib-python
 requests==2.21.0
+werkzeug<1.0.0
```

Comment:
Since we are depending explicitly we should use the debian version like other dependencies :speak_no_evil:

Reply:
Fixed thanks!